cinch/roles/check_ssh/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+ansible_port: 22
```

Comment:
With our change from raw SSH to the ansible ping kludge, is this necessary? I don't see it being used, but I could be missing it, somewhere.